frontend/src/components/Network.tsx

```diff
@@ -447,6 +593,7 @@ const Network = ({ nodes: _nodes, edges: _edges }: Props) => {
           .map((element) => element.data() as Node | Edge),
       ),
     );
+    setupCytoscapePanLimits();
```

Comment:
Very minor stylistic comment, but keeping with the naming style of the rest of the code base so far, this could just be named `setupPanLimits`, or even just `limitPan`.